src/assets/style/components/file-uploader.sass

```diff
@@ -2,61 +2,20 @@
 
 .file-uploader
```

Comment:
Would be a good idea to throw a `cursor: pointer` property here as well to communicate clickability